include/libdnf/rpm/repo.hpp

```diff
@@ -118,262 +109,349 @@ struct Repo {
     };
 
 
-    /**
-    * @brief Verify repo ID
-    *
-    * @param id repo ID to verify
-    * @return   index of the first invalid character in the repo ID (if present) or -1
-    */
-    static int verifyId(const std::string & id);
-
-    /**
-    * @brief Construct the Repo object
-    *
-    * @param id     repo ID to use
-    * @param conf   configuration to use
-    */
-    Repo(const std::string & id, std::unique_ptr<ConfigRepo> && conf, Repo::Type type = Repo::Type::AVAILABLE);
+    /// Verify repo ID
+    /// @param id repo ID to verify
+    /// @return   index of the first invalid character in the repo ID (if present) or -1
+    /// @replaces libdnf:repo/Repo.hpp:method:Repo.verifyId(const std::string & id)
+    static int verify_id(const std::string & id);
+
+    /// Construct the Repo object
+    /// @param id     repo ID to use
+    /// @param conf   configuration to use
+    /// @param base   reference to Base instance
+    /// @param type   type of repo
+    Repo(
+        const std::string & id,
```

Comment:
Minor - variable id could be renamed to repo_id

Reply:
Two examples below. Do you really want to use the second approach (repeating the class/struct name in the member names) ? Then I also must rename all classes methods. `verify_id()` -> `verify_repo_id()`, `enable()` -> `repo_enable()`, `disable()` -> `repo_disable()`, `load()` -> `repo_load()`, ... to achieve name consistency.

```
struct Person {
    int id;
    char * name; 
    int age;
};
```
```
struct Person {
    int person_id;
    char * person_name; 
    int person_age;
};

```

Reply:
Not really, id word is quite overloaded therefore I suggest to be more specific. Id, PackageId, repo_id, ReldepId, ...